misaligned.cpp

```diff
@@ -13,9 +19,29 @@ int printColorMap() {
     return i * j;
 }
 
+void TestPrintColorCodeMap() {
+    stringstream colorMapOutput;
+    streambuf* p_cout_streambuf = std::cout.rdbuf();
+    cout.rdbuf(colorMapOutput.rdbuf());
+
+    PrintColorCodeMap();
+
+    cout.rdbuf(p_cout_streambuf);
+    stringstream expectedOutput;
+    for (int i = 0; i < 5; i++) {
+        for (int j = 0; j < 5; j++) {
+            expectedOutput << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[j] << "\n";

```

Comment:
The `expectedOutput` also goes from 0 to 24? As per the standard, pair numbers need to go from 1 to 25.
Also, the column separators in the expected output are not aligned.
Try to write the test, so that "passing" gives us confidence that the solution is "acceptable to the user"